frontend/src/components/listeners/SubmitExtraction.tsx

```diff
@@ -73,6 +87,18 @@ export default function SubmitExtraction(props: SubmitExtractionProps) {
 		setActiveStep(0);
 	}
 
+	useEffect(() => {
+		const interval = setInterval(() => {
+			if (isFormSubmitted == true && job_id.length > 0) {
+				jobSummaryFetch(job_id);
+				jobUpdatesFetch(job_id);
+			}
+		}, 2000);
+
+		return () => clearInterval(interval);
```

Comment:
You might want to clear this interval when leaving this widget; Right now it's continuing calling the endpoint every two seconds even though I close the modal.

The way of clearing the interval is to store the interval in state and give it a defaut:
```	const [intervalId, setIntervalId] = useState(0);
```
Whenever you are setting the interval, make sure you update the state
```
const interval = setInterval(() => {
			if (isFormSubmitted == true && job_id.length > 0) {
				jobSummaryFetch(job_id);
				jobUpdatesFetch(job_id);
			}
		}, 2000);
setIntervalId(interval);
```

Then you can clear it this way

```
const _clear_old_timeinterval = () => {
		if (intervalId) {
			clearInterval(intervalId);
			setIntervalId(0);
		}
	};
```

Reply:
I'm not sure why but I am running the wordcount extractor, but getting nothing for logs. 

I merged main into the branch to fix the blank page error, so I hope that didn't mess something up.